### DISCUSSION TITLE
正文中的文字/公式超出边距没有自动换行

### DISCUSSION THREAD

**Discussion by yushan-li** [sjtug/SJTUThesis#1025] [提问]
在正文中写些文字或者有$$环境下的公式，有些直接超出边距没有自动换行（如附件图显示），想请教下这个可以怎么解决呀
![位置1](https://github.com/sjtug/SJTUThesis/assets/37063549/cc5822ec-b654-448d-8966-f53bf6301ea7)
![位置2](https://github.com/sjtug/SJTUThesis/assets/37063549/8ce72636-31ea-490b-83c4-b6330cff246b)

**Comment by LogCreative** [comment]
一个简单的做法是手动断一下：
```latex
\documentclass{sjtuthesis}
\usepackage{amsmath}
\begin{document}
测试测试测试测试测试测试测试测试测试测试测试测试测试测试$\bm{h}=[x_h,y_h]^\top$ $=[x+d_h\cos\theta, y+d_h\sin\theta]$
\end{document}
```
